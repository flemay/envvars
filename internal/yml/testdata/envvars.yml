tags:
  - name: tag1
    desc: desc of tag1

envvars:
  - name: ENVVAR_1
    desc: desc of ENVVAR_1

  - name: ENVVAR_2
    desc: desc of ENVVAR_2
    optional: true
    example: example1