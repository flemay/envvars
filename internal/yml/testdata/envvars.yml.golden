envvars:
- name: ENVVAR_1
  desc: desc of ENVVAR_1
